src/algebraic_topology/simplicial_type.lean

```diff
@@ -0,0 +1,74 @@
+/-
+Copyright (c) 2021 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Johan Commelin, Scott Morrison
+-/
+import algebraic_topology.simplicial_object
+import category_theory.yoneda
+
+/-!
+A simplicial type (usually called a simplicial set, when using set theoretic foundations)
+is just a simplicial object in `Type`, i.e. a `Type`-valued presheaf on the simplex category.
+
+We define the standard simplices as simplicial types, and their boundaries and horns.
+-/
+
+universes v u
+
+open category_theory
+
+/-- The category of simplicial types.
+This is the category of contravariant functors from
+`simplex_category` to `Type u`. -/
+@[derive large_category]
+def sType : Type (u+1) := simplicial_object (Type u)
+
+namespace sType
+
+/-- The `n`-th standard simplex associated with a nonempty finite linear order `n`
+is the Yoneda embedding of `n`. -/
+def standard_simplex : simplex_category ⥤ sType := yoneda
+
```

Comment:
Maybe it is already useful to define some local notation along the lines of `Δ n` or `Δ[n]` for `standard_simplex.obj n`.

Reply:
Done! I've localized the notation in`sSet`, and added a note in the module doc.